File: network/src/main/java/com/difft/android/websocket/internal/push/MismatchedDevices.java


package com.difft.android.websocket.internal.push;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class MismatchedDevices {
  @JsonProperty
  public List<Integer> missingDevices;

  @JsonProperty
  public List<Integer> extraDevices;

  public List<Integer> getMissingDevices() {
    return missingDevices;
  }

  public List<Integer> getExtraDevices() {
    return extraDevices;
  }
}

File: network/src/main/java/com/difft/android/websocket/internal/push/MismatchedDevices.kt
package com.difft.android.websocket.internal.push

import com.fasterxml.jackson.annotation.JsonProperty

data class MismatchedDevices(
    @JvmField
    @JsonProperty
    var missingDevices: List<Int> = emptyList(),

    @JvmField
    @JsonProperty
    var extraDevices: List<Int> = emptyList()
)
